addon-test-support/index.js

```diff
@@ -0,0 +1,28 @@
+import QUnit from 'qunit';
```

Comment:
We can't assume `QUnit` is available here, can you swap instead to lazily importing?